app/views/spree/admin/orders/_form/_adjustments.html.haml

```diff
@@ -0,0 +1,14 @@
+- if adjustments.present?
+  %fieldset.no-border-bottom
+    %legend= title
+    %table>
+      %thead
+        %th= Spree.t('name')
+        %th= Spree.t('amount')
+      %tbody#order-charges.with-border
```

Comment:
It is now possible for the HTML ID `order-charges` to appear more than once in the page. I see that this is also an issue with [in the 2-2-stable version of the file](https://github.com/spree/spree/blob/2-2-stable/backend/app/views/spree/admin/orders/_adjustments.html.erb), but it isn't valid HTML.

Reply:
yeah, good catch. I am not sure what to do? I feel tempted to add this to one of the tech debt issues that Pau created above. maybe #3475 ?

Reply:
Sounds good to me @luisramos0

Reply:
Hmm, it isn't tech debt though. It could get in quicker than #3475, so I would create a new issue for it. :slightly_smiling_face: I'll do that in a bit.

Reply:
I just added this to #3477
both your points need to be fixed now @kristinalim they are not tech debt.